### PR TITLE
[EvaluatorFactory] Add tests

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

File: octobot_evaluators/evaluators/evaluator_factory.py [hunk 1]
```diff
@@ -64,26 +64,28 @@ async def create_evaluators(evaluator_parent_class,
...
+    return crypto_currency_name_by_crypto_currencies[cryptocurrency] \
+        if crypto_currency_name_by_crypto_currencies \
+           and cryptocurrency is not None \
+           and not evaluator_class.get_is_cryptocurrency_name_wildcard() \
+        else None
 
 
 def _get_cryptocurrencies_to_create(evaluator_class, crypto_currency_name_by_crypto_currencies):
-    return crypto_currency_name_by_crypto_currencies.keys() \
+    return list(crypto_currency_name_by_crypto_currencies.keys()) \
```
Comment (GuillaumeDSM) [inline octobot_evaluators/evaluators/evaluator_factory.py:75]: :+1:

File: tests/__init__.py [hunk 2]
```diff
@@ -13,14 +13,78 @@
...
 async def matrix_id():
     created_matrix_id = create_matrix()
     yield created_matrix_id
-    Matrices.instance().del_matrix(created_matrix_id)
+    matrices.Matrices.instance().del_matrix(created_matrix_id)
+
+
+@pytest.yield_fixture
+async def install_tentacles():
+    def _download_tentacles():
```
Comment (GuillaumeDSM) [inline tests/__init__.py:58]: Is it necessary?

**Review by GuillaumeDSM — COMMENTED** [review]
Very good job !

**Review by GuillaumeDSM — APPROVED** [review]

File: tests/__init__.py (hunk 2 above)
Comment (Herklos) [reply tests/__init__.py:58]: Not anymore :+1:

**Pull request merged**